netcontrold/lib/dataif.py

```diff
@@ -618,6 +618,8 @@ def pmd_load_variance(pmd_map):
     pmd_map : dict
         mapping of pmd id and its Dataif_Pmd object.
     """
+    update_pmd_load(pmd_map)
```

Comment:
The caller has to ensure updating pmd load before calling this function. Discard this change.